schema_based_on_diagram.sql

```diff
@@ -0,0 +1,45 @@
+create DATABASE clinic;
+
+
+create table patients (
+    id serial PRIMARY KEY,
+    name varchar(50),
+    date_of_birth date
+);
+
+create table invoices (
+    id serial PRIMARY KEY,
+    generated_at date,
+    payed_at date,
+    medical_histories_id integer references medical_histories(id)
+);
+
+create table treatments (
+    id serial PRIMARY KEY,
+    type varchar(50),
+    name varchar(50)
+);
+
+create table medical_histories (
+    id serial PRIMARY KEY,
+    admitted_at date,
+    patient_id integer references patients(id),
+    status varchar(50)
+);
+
+create table histories_tratments (
+    medical_histories_id integer references medical_histories(id),
+    treatments_id integer references treatments(id)
```

Comment:
Great job team 🤝

- [ ] As stated in the [project requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/database_from_diagram.md), kindly ensure all foreign keys have indexes created.